atlp-rwanda/rca-phantom-team1-bn
# Chore do registration

### **What does this PR do**
This pull request implementation of registering new drivers and operators.

### **Description**

1. The administrator should be able to register a new driver or a new operator.
2. An email containing a random password should be sent to the provider email.
3. The driver or operator should be able to sign in to the app using their email and the provided password.
4.Appropriate error messages are displayed on unsuccessful attempts.

### **How to manually test this**

1. Check in the branch chore-do-registration
2. Run the server
3. Open the Swagger API documentation to test driver & operators

### **Relevant Trello story**
[10-register-drivers-operators][(https://trello.com/c/SDfWF0QF/10-register-drivers-operators)](https://trello.com/c/SDfWF0QF/10-register-drivers-operators)

### **Screenshots** 

![result](https://user-images.githubusercontent.com/54369909/228188765-a6eed883-1686-4169-ae21-ab1dc32259c4.png)
![phantom](https://user-images.githubusercontent.com/54369909/228188781-d71647c6-2603-4f60-8f79-e6196b356d46.png)
